test/unittest/core_deployment_test.yaml

```diff
@@ -0,0 +1,547 @@
+suite: CoreDeployment
+
+tests:
+  - it: PodLabels
+    set:
+      core:
+        podLabels:
+          test.label: test-label
+    template: templates/core/core-dpl.yaml
+    asserts:
+      - equal:
+          path: spec.template.metadata.labels["test.label"]
+          value: test-label
+
+  - it: PodAnnotations
+    set:
+      core:
+        podAnnotations:
+          test.annotation: test-annotation
+    template: templates/core/core-dpl.yaml
+    asserts:
+      - equal:
+          path: spec.template.metadata.annotations["test.annotation"]
+          value: test-annotation
+
+  - it: PodAnnotations
+    set:
+      core:
+        podAnnotations:
+          test.annotation: test-annotation
+    template: templates/core/core-dpl.yaml
+    asserts:
+      - equal:
+          path: spec.template.metadata.annotations["test.annotation"]
+          value: test-annotation
```

Comment:
line 26 - line35 is a duplicate of line 15 - line24

Reply:
Removed the duplicated tests and renamed the wrong ones.